resources/views/statistics/index.blade.php

```diff
@@ -45,7 +57,17 @@
                         </table>
                     </div>
                     <div class="card-footer">
-                        *User with at least 1 elePHPant in herd.
+                        <p class="mb-0">
+                            *User with at least 1 elePHPant in herd.
+                        </p>
+                        @if (auth()->check())
+                        <p class="mb-0">
+                            <i class="fa-solid fa-check mr-2 text-success"></i> = Elephpant exists in your collection
+                        </p>
+                        <p class="mb-0">
+                            <i class="fa-solid fa-x mr-2 text-danger"></i> = Elephpant does not exist in your collection
```

Comment:
please change Elephpant to elePHPants so it is equal :-)

Reply:
Changed to be correct! Thanks!